lib/SPIRV/SPIRVLowerMemmove.cpp

```diff
@@ -105,6 +125,7 @@ class SPIRVLowerMemmove: public ModulePass,
   static char ID;
 private:
   LLVMContext *Context;
+  Module *Mod;

```

Comment:
line 90 to query the size of the allocated type: `Mod->getDataLayout()->getTypeSizeInBits(SrcTy->getPointerElementType())`. I created the member, since I didn't know how else to get the Module instance.